config.yml

```diff
@@ -9,6 +9,7 @@ default:
     - pandas==1.5.3
     - ase==3.22.1
     - ipywidgets
+    - mat3ra-standata
```

Comment:
Let's add to where it's required only

Reply:
But if it's installed everywhere, it's faster to import and use in any notebook. you don't need to specify this, just import standata and use